src/main/java/org/broadinstitute/consent/http/service/DacService.java

```diff
@@ -306,13 +306,13 @@ private boolean isAuthUserChairOrMember(AuthUser authUser) {
     /**
      * Filter data access requests by the DAC they are associated with.
      */
-    List<DataAccessRequest> filterDataAccessRequestsByDac(List<DataAccessRequest> documents, AuthUser authUser) {
-        if (isAuthUserAdmin(authUser)) {
+    List<DataAccessRequest> filterDataAccessRequestsByDac(List<DataAccessRequest> documents, User user) {
+        if (DarUtil.hasUserRole(user, 4)) {
```

Comment:
Same feedback as below, either mimic `isAuthUserAdmin` or reference the enum values instead of using integer values.